src/USER-FEP/pair_lj_charmm_coul_long_soft.cpp

```diff
@@ -688,41 +688,23 @@ void PairLJCharmmCoulLongSoft::init_style()
     error->all(FLERR,
                "Pair style lj/charmm/coul/long/soft requires atom attribute q");
 
-  // request regular or rRESPA neighbor lists
+  // request regular or rRESPA neighbor list
 
   int irequest;
+  int respa = 0;
 
   if (update->whichflag == 1 && strstr(update->integrate_style,"respa")) {
```

Comment:
while you are modifying these files, instead of using `strstr()` this test should be using `utils::strmatch(update->integrate_style,"^respa")`